content/concepts/tokens/common-misconceptions-about-freezes.md

```diff
@@ -0,0 +1,32 @@
+---
+parent: freezes.html
+blurb: Clarify common misunderstandings about the XRP Ledger's freeze feature.
+labels:
+  - Tokens
+---
+# Common Misunderstandings about Freezes
+
+It is a common misconception that Ripple or others can freeze XRP, similar to how centralized services like PayPal can suspend your account and prevent you from accessing your funds. In reality, while the XRP Ledger does have a [freeze feature](freezes.html), it can only be used on issued tokens, not on XRP. **No one can freeze XRP.**
```

Comment:
Should we mention the XRP Ledger Foundation in addition to Ripple?

Reply:
The common misconception is specifically about Ripple, so I don't think it warrants a mention as of now.